plan/physical_plans.go

```diff
@@ -1173,8 +1173,18 @@ func buildSchema(p PhysicalPlan) {
 	switch x := p.(type) {
 	case *Limit, *TopN, *Sort, *Selection, *MaxOneRow, *SelectLock:
 		p.SetSchema(p.Children()[0].Schema())
-	case *PhysicalHashJoin, *PhysicalIndexJoin:
+	case *PhysicalHashJoin:
 		p.SetSchema(expression.MergeSchema(p.Children()[0].Schema(), p.Children()[1].Schema()))
+	case *PhysicalIndexJoin:
```

Comment:
This logic is the same as MergeJoin ?

Reply:
No, there is a little difference when the join type is `InnerJoin`, `LeftOuterJoin` and `RightOuterJoin`